File: Java/1295_Find_Numbers_with_Even_Number_of_Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int result = 0;
        
        for(int num : nums){
            if((num / 10 > 0 && num / 10 < 10) || (num / 1000 > 0 && num / 1000 < 10) || (num / 100000 > 0 && num / 100000 < 10))
            {
                result++;
            }
        }
        
        return result;
    }
}

File: CS/1295_Find_Numbers_with_Even_Number_of_Digits.cs
public class Solution {
    public int FindNumbers(int[] nums) {
        int result = 0;
        foreach(int num in nums){
            if((num == 100000) || (num < 10000 && num >= 1000) || (num < 100 && num >= 10)) 
            {
                result++;
            }
        }
        return result;
    }
}
